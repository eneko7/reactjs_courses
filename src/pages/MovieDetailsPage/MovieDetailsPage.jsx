import React from 'react';
import MovieDetails from '../../components/MovieDetails';
import ResultLine from '../../components/ResultLine';
import MoviesGrid from '../../components/MoviesGrid';
import Footer from '../../components/Footer';
import ErrorBoundary from '../../components/ErrorBoundary/ErrorBoundary';

const MovieDetailsPage = () => (
  <ErrorBoundary>
    <MovieDetails />
    <ResultLine />
    <MoviesGrid />
    <Footer />
  </ErrorBoundary>
);

export default MovieDetailsPage;
