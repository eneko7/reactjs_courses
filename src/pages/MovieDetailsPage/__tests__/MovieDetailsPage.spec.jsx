import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import MovieDetailsPage from '../MovieDetailsPage';

const renderer = new ShallowRenderer();
describe('MovieDetailsPage', () => {
  it('renders correctly', () => {
    renderer.render(<MovieDetailsPage />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
