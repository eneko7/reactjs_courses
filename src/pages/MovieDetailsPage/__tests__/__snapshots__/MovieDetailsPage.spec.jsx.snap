// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetailsPage renders correctly 1`] = `
<ErrorBoundary>
  <withRouter(Connect(MovieDetails)) />
  <withRouter(Connect(ResultLine)) />
  <Connect(MoviesGrid) />
  <Footer />
</ErrorBoundary>
`;
